### ISSUE TITLE
the chainA  of design_ppi_X.pdb file in output all is GGGGGG....

### ISSUE THREAD

**Issue by RyanYip-Kat** [RosettaCommons/RFdiffusion#299]
Is it normal that the chain a in the design_ppi_ X.pdb file of the output result is GGGGG..?

run command:
python ./scripts/run_inference.py inference.output_prefix=example_outputs/design_ppi inference.input_pdb=examples/input_pdbs/insulin_target.pdb 'contigmap.contigs=[A1-150/0 70-100]' 'ppi.hotspot_res=[A59,A83,A91]' inference.num_designs=10 denoiser.noise_scale_ca=0 denoiser.noise_scale_frame=0

**Comment by roccomoretti** [comment]
Yes. RFdiffusion only generates backbone structures. As such, the output at positions which were generated by RFdiffusion will by Gly (i.e. only a backbone).

If you do want to determine what the sequence at those positions should be, use a structure to sequence prediction model like [ProteinMPNN](https://github.com/dauparas/ProteinMPNN) on the generated backbones.

**Comment by RyanYip-Kat** [comment]
Okay!Thanks
